### ISSUE TITLE
In admin select field selected choice displays as html

### ISSUE THREAD

**Issue by dbrabyn** [nlemoine/acf-country#48]
Details
-------

 - WordPress version: 5.7.2
 - ACF version: Pro 5.9.8
 - For UX/UI issues: MacOS, Safari v 14.1.1 and Chrome v 91.0.4472.114 (Official Build) (x86_64)
 - Allow null: true
 - Multiple values: false
 - Stylised UI: true
 - Condtional logic: false

Reproduction
------------

### Bug summary

In the admin the dropdown list displays the flag icons and country names correctly but the select field shows the following html for the flag icon and country name (in this case South Africa).
`<span class="acf-country-flag-icon famfamfam-flags za"></span> <span class="acf-country-flag-name">Afrique du Sud</span>`

### Steps to reproduce

Regular WP install on local and production sites.

### Actual result

See screen grab:
![CleanShot 2021-07-16 at 18 19 23@2x](https://user-images.githubusercontent.com/6307031/126013985-76257dc1-4780-401d-af1f-9b310b9101fe.jpg)

### Expected result

In the admin the select field should display the flag icon and country name -- as it does for the dropdown list .

**Comment by nlemoine** [comment]
Hello @dbrabyn 

I'm not able to reproduce your bug. Do you have other plugins activated on your site? Maybe another plugin is loading a different version of select2. Try deactivating other plugins.

**Comment by stirrell** [comment]
I was just having the same issue. In my case, the culprit was WooCommerce. Toggling WooCommerce causes this issue to go away/come back.

**Comment by dbrabyn** [comment]
Yep, WooCommerce is loading plugins/woocommerce/assets/js/select2/select2.full.min.js?ver=4.0.3 which is really [selectWoo](https://github.com/woocommerce/selectWoo), "a fork of select2 with improved accessibility".

**Comment by zuperuser** [comment]
Having the same issue.
WPML is loading select2 too (and I can't de-activate this plugin...).
How can we solve this problem?

**Comment by nlemoine** [comment]
https://www.advancedcustomfields.com/blog/acf-5-10-release-html-escaping-blocks-api-v2-block-preloading-and-more/

A new version of ACF has been released:

> Update: We’ve also pushed out version 5.10.1, which includes a fix for a conflict with WooCommerce’s SelectWoo library.

Can you tell if that fixes the issue?

**Comment by dbrabyn** [comment]
I'm still seeing it with ACF Pro 5.10.1.

**Comment by zuperuser** [comment]
Same here; still HTML in the dropdowns...

**Comment by lgladdy** [comment]
Hey everyone,

This should be resolved in ACF 5.10.2, but please let me know if not!

It seems to be working okay for me now. Try clearing your browser caches if you still have issues.

![Screenshot 2021-09-01 at 13 51 20](https://user-images.githubusercontent.com/213440/131674250-2834a7c8-4695-4c08-a99a-12a75e99a20f.png)

**Comment by dbrabyn** [comment]
Still there for me.

**Comment by nlemoine** [comment]
Hi ACF-country users,

I'm thinking about removing flags images in favor of [emoji flags](https://gist.github.com/chrisjonesio/2f693fd9853c5450345e18c836577393): lighter, removing a dependency, no more select2 issues, etc.

What do you think about it?

**Comment by zuperuser** [comment]
ACF 5.10.2 solved it for me.
BTW: emoji flags are a good idea!

**Comment by hercilioseox** [comment]
Hey there. Just found the same issue.
ACF: 5.11.4
WooCommerce: 6.1.1 
ACF Country: 2.0.2

The [emoji flags](https://gist.github.com/chrisjonesio/2f693fd9853c5450345e18c836577393) idea looks great. Please let us know when we can test it.

**Comment by nlemoine** [comment]
Looks like emoji flags are dead 😢 

https://blog.unicode.org/2022/03/the-past-and-future-of-flag-emoji.html

That's a shame and a bad coincidence, a few after days after I almost finished writing a new version with emoji flags...

**Comment by EmmanuelBeziat** [comment]
> Looks like emoji flags are dead 😢

Ah dang, I was just coming to say I have the bug too (WP 6.0 / ACF 5.11 / WooCommerce 6.5.1), and that emojis was an awesome idea.

Too bad :(

**Comment by nlemoine** [comment]
@EmmanuelBeziat However, countries doesn't disappear or show up every morning (although the recent events may remind us that borders are not that set in stone 😞)

I think I'll still push this new update soon.

**Comment by nlemoine** [comment]
Finally got a project where I needed to use ACF Country so I decided to give it an update. 

You can try it here: https://github.com/nlemoine/acf-country/tree/3.x

I finally went for using emojis flags although [they're pretty dead](https://github.com/nlemoine/acf-country/issues/48#issuecomment-1118265151) but it provides the huge benefit of ending those Select2 issues. It's also lightweight and faster to show since you can display flags without the need to even use Select2.

Note that it's ACF 6 compatible for now, at least if you use ACF UI.
